static/js/components/AssignmentForm.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useForm, Controller } from 'react-hook-form';
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import FormControl from "@material-ui/core/FormControl";
+import MenuItem from "@material-ui/core/MenuItem";
+import { makeStyles } from "@material-ui/core/styles";
+import * as Actions from '../ducks/source';
+
+import 'react-datepicker/dist/react-datepicker-cssmodules.css';
+
+
+function makeMenuItem(observingRun, instrumentList, telescopeList, groups) {
+  const { instrument_id } = observingRun;
+  const instrument = instrumentList.filter((i) => i.id === instrument_id)[0];
+  const instundef = instrument === undefined;
+
+  const telescope_id = !instundef ? instrument.telescope_id : undefined;
```

Comment:
Can do away with instundef above and just check the truthy value of instrument here